app/src/components/report/ReportCardView.tsx

```diff
@@ -0,0 +1,112 @@
+import { IReport } from '@/models';
+import Card from 'react-bootstrap/Card';
+import Stack from 'react-bootstrap/Stack';
+import { Typography, Grid } from '@mui/material';
+import {
+    StyledReportDetails,
+    Box1,
+    Heading,
+    FooterDetails,
+    IDText,
+    Box2,
+} from '@/styles/StyleCardView';
+
+export type ReportCardProps = {
+    report: IReport;
+};
+
+/** Report Card View */
+export default function ReportCard({ report }: ReportCardProps) {
+    return (
+        <Card>
+            <StyledReportDetails>
+                <Grid container spacing={2}>
+                    <Grid item xs={6}>
+                        <Box1>
+                            <Heading variant="h6">Contact Information</Heading>
```

Comment:
Non-blocking: is `h6` correct here? Please correct me if I'm wrong, but I don't think we use h3, h4, h5. Using the correct heading level is useful to organize information on our page hierarchically, and important to screenreaders

Often, devs will use different heading levels depending on what "looks right," but we should use the correct heading level and use CSS to make it look right :)